examples/resources/btp_subaccount_environment_instance/resource.tf

```diff
@@ -35,3 +35,26 @@ resource "btp_subaccount_environment_instance" "cloudfoundry" {
     delete = "30m"
   }
 }
+
+# creates a kyma environment in a given account
+# NOTE: for the available parameter values, check https://help.sap.com/docs/btp/sap-business-technology-platform/available-plans-in-kyma-environment
+resource "btp_subaccount_environment_instance" "kyma" {
+  subaccount_id    = "6aa64c2f-38c1-49a9-b2e8-cf9fea769b7f"
+  name             = "my-kyma-environment"
+  environment_type = "kyma"
+  service_name     = "kymaruntime"
+  plan_name        = "aws"
+
+  parameters = jsonencode({
+    name            = "my-kyma-environment"
```

Comment:
do you need to specify the name twice?

Reply:
According to the docu it is display-name and cluster name. They can differ.